src/D2L.CodeStyle.TestAnalyzers/Common/NotNullWhenAttribute.cs

```diff
@@ -0,0 +1,14 @@
+﻿﻿#if !NETSTANDARD2_1
+
+namespace System.Diagnostics.CodeAnalysis;
+
+[AttributeUsage( AttributeTargets.Parameter )]
+internal sealed class NotNullWhenAttribute : Attribute {
+	public bool ReturnValue { get; }
+
+	public NotNullWhenAttribute( bool returnValue ) {
+		ReturnValue = returnValue;
+	}
+}
+
+#endif

```

Comment:
The other project brings in a package for this but I'm pretty meh on that.

In the future I'm going to make a folder for common source for analyzers and some Directory.Build.targets magic to `<Compile Include="..." LinkBase="Common" />`-in some shared analyzer code and this can go in there.